oncall/test-infra.yaml

```diff
@@ -0,0 +1,48 @@
+args:
+- --victims=fejta,ixdy,rmmh,krzyzacy,cjwagner
```

Comment:
:joy:

Reply:
i.e. add @BenTheElder

Reply:
It's not Q4 yet. I still need a week to finish the majority of Jenkins -> Prow :smiling_imp: